commons-hpcc/src/main/java/org/hpccsystems/commons/benchmarking/MetricAverageTransformer.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package org.hpccsystems.commons.benchmarking;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+
+/**
+ * Helper class that combines metrics with the same name into an AveragedMetric.
+ */
+public class MetricAverageTransformer
+{
+    public List<IMetric> transform(List<IMetric> metrics)
```

Comment:
is this meant to override IMetricSetTransformer.transform()?
If so, annotate with  @Override